### DISCUSSION TITLE
Recommended price : max 15$/unit

### DISCUSSION THREAD

**Discussion by nliaudat** [nliaudat/esp32_8ch_motor_shield#38] [Buy or Sell]
The last I bought cost 55.71 USD for 5 units shipping included = approx 11$ per unit

[Prices at 2023-03-04]

If the voltage regulator is included, the good price is 15$. 
If Esp32 +3.5$

If you buy more (10x), you can lower the price

**Please do not make profit on that project !!!** It's no commercial use licence

**Comment by WMP** [comment]
You wrote this thread because I posted links to my offers at 130 pln (today $30.66 ) and 140 pln (today 33.02 ). You deleted the thread.  I already explain why such a price. Well, in addition to the PCBs designed by you, I also bought (and in fact my colleague ordered and paid, I refund him part of the cost) others for other projects - the purchase price exceeded 150 euros. According to EU law, if the purchase from China exceeds 150 euros, you have to charge it with "Delivery Duty Paid". This means that all customs duties and taxes are raised by the seller - jlcpcb. In the case of IOSS, the seller pays only VAT - 23% in Poland. Customs duty of 2.3% for electronic components (not PCBs) + customs work fees is paid by the buyer. 

Today your complete PCB (without HLK-5M03) in DDP option costs 191.85 USD (I paid 16.03.2023 1 USD = 4.4267 so 191.85 USD = 849.26 PLN):
![image](https://user-images.githubusercontent.com/45381/232607538-f58d3041-7e6a-4aec-b570-e07c2c9ef076.png)

For the IOSS option, the cheapest shipping costs 160.69 USD 
![image](https://user-images.githubusercontent.com/45381/232608485-e644156b-6937-4bab-bb02-5f4447b2c1b5.png)

But, I due to the size of the order was forced to choose the DDP option. 

That is, for 10 PCBs without power supply I paid 849.26 PLN . I had to buy 10 power supplies: https://allegro.pl/oferta/hlk-5m03-zasilacz-240-3-3v-1500ma-1-5a-do-druku-10766791996 , that is 21 PLN * 10 = 210 PLN (today 49,54 USD) without shipping - I have it within the subscription for 45 PLN per year.

In total we already have: 1059,26 PLN. On top of that, all the ESP32 I have at home are too wide and I need to add one goldpin 1x20 socket : https://allegro.pl/oferta/gniazdo-szpilkowe-proste-na-goldpin-1x20-2szt-7366944616 - the offer is for two pieces, so one piece costs 1.17 PLN * 10 = 11,70 PLN.
I still want to add Freepins 2x6 pin sockets : https://allegro.pl/oferta/piny-goldpin-1x6-gniazdo-proste-5szt-1336-12707260781 - offer is for 5 pieces, I need such 20 pieces - total 7,96 PLN.
 
Let's sum up:
 - assembled PCB including shipping: 849,26 PLN
 - power supplies: 210 PLN
 - Additional sockets if the buyer wants them: 11,70 PLN + 7,96 PLN 
TOTAL: 1079.66 PLN (today 254.67 USD). **That is, one PCB ultimately costs 107 PLN / 25.47 USD**.

I posted them on Allegro for 140PLN (from selling one tile I have to pay 7.9% = 11.06 ) and on OLX for 130 PLN (here there is a lower fee). However, thanks to the fact that I decided in this post to carefully count the costs, I reduced the price by 10 PLN so that no one would accuse me of trying to make money. If I sell something, I will earn about 13 PLN on it, or 3 bread.

**Comment by nethawk2** [comment]
Hello,
i write only to show, how much the boards did cost me in 2022.
i ordered at JLCPCB january 2022. The boards are assembled, but without an ESP.
![jlcpcb2022price](https://user-images.githubusercontent.com/52479299/232742522-1e5c565e-3f41-4f9e-bac7-965ca6afb227.jpg)

Greetings Torsten

**Reply by WMP** [reply]
So 18 USD per board.

**Comment by nliaudat** [comment]
![image](https://user-images.githubusercontent.com/6782613/232839499-c33e86fe-517e-4b87-ae79-1927317bf380.png)

= 11.15$/board

**Reply by WMP** [reply]
Why is your shipping so cheap, and why don't you have a duty charge?

**Reply by nliaudat** [reply]
I use global shipping (15-20 days) and if order is less than 65$, there is no duty charge for Switzerland

**Reply by nstadigs** [reply]
I think you are kinda privileged when it comes to this. Personally I would love to a couple of boards for $15 since the cheapest configuration from JLCPCB for five boards with "global shipping" to Sweden , without the power supplies is $107.53. Thats $21.5 per board, plus power supply.

This is for the latest RJ11 version tho, so I don't have to alter the cables. I also created a personal box for the RJ11 version with holes for the contacts.

**Comment by heinemannj** [comment]
Looking for 1x assembled PCB v57 rev 1.48 or newer
- with assembled SMT parts, RJ11 / screw terminal
- with custom box
- with ESP
- delivery to Germany

Any offers?

**Comment by marcelveldt** [comment]
Hi, I'm interested in ordering 2 units (preferred pre-assembled but can live with some soldering on my own if I have to).
If possible with a housing (or tips what housing could potentially fit). Shipping to the Netherlands.

**Reply by Ceglarek10** [reply]
Hallo. Ich werde 5 stk bestücken lasse. Brauche 2. Wer interesse an 3 hat, bitte schreiben.

**Reply by nliaudat** [reply]
please open a new discussion. 
That is from 2023
